docs/swarm/admin_guide.md

```diff
@@ -0,0 +1,246 @@
+`--force-new-cluster`.
+
+You manage cluster membership with the `docker swarm` and `docker node`
+subsystems. Refer to [Add nodes to a swarm](join-nodes.md) for more information
+on how to add worker nodes and promote a worker node to be a manager.
+
+## Distributing manager nodes
+
+In addition to maintaining an odd number of manager nodes, pay attention to
+datacenter topology when placing managers. For optimal fault-tolerance, distribute
+manager nodes across a minimum of 3 availability-zones or datacenter
+regions to support failures of an entire set of machines or common maintenance
+scenarios. If you suffer a failure in any of those zones, the swarm should maintain
+a quorum of manager nodes available to process requests and rebalance workloads.
+
+> Warning: Distributing managers between regions could be slow depending on the
+> infrastructure used to deploy the nodes. Write latencies could be high and issues
+> could arise with Leader election and Health Checks.
+
+| Swarm manager nodes |  Repartition (on 3 regions)  |
+|:-------------------:|:----------------------------:|
+| 3                   |            1-1-1             |
+| 5                   |            2-2-1             |
+| 7                   |            3-2-2             |
+| 9                   |            3-3-3             |
+
+## Run manager-only nodes
+
+By default manager nodes also act as a worker nodes. This means the scheduler
+can assign tasks to a manager node. For small and non-critical clusters
+assigning tasks to managers is relatively low-risk as long as you schedule
+services using **resource constraints** for *cpu* and *memory*.
+
+However, because manager nodes use the Raft consensus algorithm to replicate data
+in a consistent way, they are sensitive to resouce starvation. You should
+isolate managers in your swarm from processes that might block cluster
+operations like cluster heartbeat or leader elections.
+
+To avoid interference with manager node operation, you can drain manager nodes
+to make them unavailable as worker nodes:
+
+```bash
+docker node update --availability drain <NODE-ID>
+```
+
+When you drain a node, the scheduler reassigns any tasks running on the node to
+other available worker nodes in the cluster. It also prevents the scheduler from
+assigning tasks to the node.
+
+## Back up the cluster state
+
+Docker manager nodes store the cluster state and manager logs in the following
+directory:
+
+`/var/lib/docker/swarm/raft`
+
+Back up the raft data directory often so that you can use it in case of disaster
+recovery.
+
+You should never restart a manager node with the data directory from another
+node (for example, by copying the `raft` directory from one node to another).
+The data directory is unique to a node ID and a node can only use a given node
+ID once to join the swarm. (ie. Node ID space should be globally unique)
+
+To cleanly re-join a manager node to a cluster:
```

Comment:
this set of steps does not seem to apply to "backing up the cluster state" - does it deserve its own section?